Sources/BraintreePayPal/BTPayPalVaultRequest.swift

```diff
@@ -1,53 +1,23 @@
 import Foundation
 
-#if canImport(BraintreeCore)
-import BraintreeCore
-#endif
-
 ///  Options for the PayPal Vault flow.
-@objcMembers open class BTPayPalVaultRequest: BTPayPalRequest {
+@objcMembers public class BTPayPalVaultRequest: BTPayPalVaultBaseRequest {
 
     // MARK: - Public Properties
 
-    /// Optional: Offers PayPal Credit if the customer qualifies. Defaults to `false`.
-    public var offerCredit: Bool
+    /// Optional: Used to determine if the customer will use the PayPal app switch flow.
+    /// Defaults to `false`.
+    /// - Note: This property is currently available in limited release
```

Comment:
Take it or leave it - Should we a add the note that it's a "beta" and could be removed?